chromadb/api/client.py

```diff
@@ -136,14 +140,24 @@ def count_collections(self) -> int:
     def create_collection(
         self,
         name: str,
-        configuration: Optional[CollectionConfiguration] = None,
+        configuration: Optional[CreateCollectionConfiguration] = None,
         metadata: Optional[CollectionMetadata] = None,
         embedding_function: Optional[
             EmbeddingFunction[Embeddable]
         ] = ef.DefaultEmbeddingFunction(),  # type: ignore
         data_loader: Optional[DataLoader[Loadable]] = None,
         get_or_create: bool = False,
     ) -> Collection:
+        if configuration is None:
+            configuration = {}
+            if metadata is not None:
+                configuration = (
+                    create_collection_configuration_from_legacy_collection_metadata(
```

Comment:
this transform happens on the rust side, I don't think we need to worry about it here